### PR TITLE
Welding sounds & messages

### PR DESCRIPTION
**Pull request by EvenInDeathIStillServe**
This PR does 3 things:

- Adds a message and sound effect (from /tg/) when toggling welding tools.
- Adds 4 new sound effects for welding.
~~Makes tool worksounds lists, so more than 1 sound effect can be added, and will be randomly played.~~ _Reverted because there was a much cleaner way to do this._
- Adds a new list for welding sounds, the worksound is randomly picked from that list. Also works for repeating sounds.

### CODE REVIEW

**Comment by EvenInDeathIStillServe** [comment]
> This isn't making the most of the new sounds. It needs to randomise the sound file for each time it plays, not just once at the start
> 
> Look at zmethods for how to do this

Done. 

I reverted the change I made that turned the worksounds into lists, since it wasn't necessary. Also, made it so that welding tools spawn sparks throughout the whole task instead of just the start.

File: code/game/sound.dm [hunk 1]
```diff
@@ -95,6 +95,13 @@ var/list/climb_sound = list(
 	'sound/effects/ladder4.ogg'
 )
 
+var/list/weld_sound = list(
+	'sound/items/welding1.ogg',
```
Comment (NanakoAC) [inline code/game/sound.dm]: More variety is good but there's nothing wrong with the old welder sound, add it to this list

Comment (EvenInDeathIStillServe) [reply code/game/sound.dm]: Have you listened to the sounds? They are very different from the old sound, it would seem out of place.

Comment (NanakoAC) [reply code/game/sound.dm]: Yes i have, I disagree. Requested change stands. please add it back in that list

**Comment by EvenInDeathIStillServe** [comment]
Added the old sound to the list.

**Pull request merged**